src/rpdk/core/contract/resource_client.py

```diff
@@ -277,7 +347,7 @@ def call_and_assert(
             error_code = self.assert_failed(status, response)
         return status, response, error_code
 
-    def call(self, action, request):
+    def call(self, action, request, primary_identifier_path=None):
```

Comment:
why is this needed? isnt the primary identifier an object member? no reason to pass it in if so